### PR TITLE
Added formula for pict

### PR DESCRIPTION
**Pull request by glsorre**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
Could you ask upstream to tag a version? It's required to be accepted by homebrew as defined in: "Stable versions" in https://docs.brew.sh/Acceptable-Formulae

**Comment by glsorre** [comment]
I have done it.

**Comment by glsorre** [comment]
Hi a version (3.7.1) han been added upstream. Hope we can proceed now.

File: Formula/pict.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Pict < Formula
+  desc "Pairwise Independent Combinatorial Tool"
+  homepage "https://github.com/Microsoft/pict/"
+  url "https://github.com/Microsoft/pict/archive/v3.7.1.tar.gz"
+  sha256 "4fc7939c708f9c8d6346430b3b90f122f2cc5e341f172f94eb711b1c48f2518a"
+
+  def install
+    system "make"
+    bin.mkpath
+    bin.install "pict"
```
Comment (fxcoudert) [inline Formula/pict.rb]: `make install` to install other files than the binary? doc, etc

File: Formula/pict.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  end
+
+  test do
+    system "test", "#{bin}/pict"
```
Comment (fxcoudert) [inline Formula/pict.rb]: We need a test that exercises the some of the functionality of the app.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/pict.rb (hunk 1 above)
Comment (glsorre) [reply Formula/pict.rb]: It is a very simple cli app all the info are given launching it without parameters.

File: Formula/pict.rb (hunk 2 above)
Comment (glsorre) [reply Formula/pict.rb]: After that line I effectively test that the cli command runs coorectly.

Comment (fxcoudert) [reply Formula/pict.rb]: Just check that it outputs its own name is not sufficient. We need to exercise some functionality please.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/pict.rb (hunk 2 above)
Comment (glsorre) [reply Formula/pict.rb]: ok...I will improve this

Comment (glsorre) [reply Formula/pict.rb]: Improved

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/pict.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    sha256 "ac5e3561f9c481d2dca9d88df75b58a80331b757a9d2632baaf3ec5c2e49ccec"
+  end
+
+  def install
+    system "make"
+    bin.install "pict"
+  end
+
+  test do
+    variables = resource("testfile").fetch
```
Comment (fxcoudert) [inline Formula/pict.rb]: Use `resource("testfile").stage testpath`.
Then `output = shell_output("#{bin}/pict pict.txt")`

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @glsorre for the pull request!

**Pull request closed without merging**